src/style.css

```diff
@@ -0,0 +1,53 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  min-height: 100%;
+  overflow: hidden;
```

Comment:
For what this properties? How it helps to solve your task. 
Explain (in fe_chat) or remove redundant properties